code/__DEFINES/devices.dm

```diff
@@ -18,3 +18,17 @@
 
 // Used by PDA and cartridge code to reduce repetitiveness of spritesheets
 #define PDAIMG(what) {"<span class="pda16x16 [#what]"></span>"}
+
+//N-spect scanner defines
+#define INSPECTOR_PRINT_SOUND_MODE_NORMAL 1
+#define INSPECTOR_PRINT_SOUND_MODE_CLASSIC 2
+#define INSPECTOR_PRINT_SOUND_MODE_HONK 3
+#define INSPECTOR_PRINT_SOUND_MODE_FAFAFOGGY 4
+#define INSPECTOR_PRINT_SOUND_MODE_FAFAFOGGY_ALT 5
```

Comment:
Can you remove the fafafoggy sounds and modes?

Reply:
Changed.